src/content/release/breaking-changes/flutter-gradle-plugin-apply.md

```diff
@@ -182,6 +182,16 @@ very top of your file, add:
 +}
 ```
 
+Finally, if your `dependencies` sections contains a dependency on `"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"`, 
```

Comment:
Id change it to "if your `dependencies` block" to not potentially confuse readers with same word for block/section